### PR TITLE
[t1832] Added normalization

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5dab3f13e6ca90c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5dab3f13e6ca90c/output.txt

Total script time: 1.14 mins

Server is running on http://pr668.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/32d288c7628eedd/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/32d288c7628eedd/output.txt

Total script time: 0.98 mins

File: css/globals.less [hunk 1]
```diff
@@ -1,9 +1,12 @@
-// VARIABLES
+// SPACING
 @HEADER_SPACING: 50px;
 @TRAY_HEIGHT: 220px;
 @TRAY_SPACING: @TRAY_HEIGHT - 10;
 @TRAY_SPACING_MINIMIZED: 35px;
 
+//COLORS
```
Comment (mjschranz) [inline css/globals.less]: Why no space here after the comment slashes here, but in other spots?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d19f71d6be12eb2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d19f71d6be12eb2/output.txt

Total script time: 1.57 mins

Server is running on http://pr668.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d0cde6846f975d2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d0cde6846f975d2/output.txt

Total script time: 1.47 mins

Server is running on http://pr668.butyr.org/templates/test.html

**Pull request merged**